src/pages/History/HistoryTable/index.tsx

```diff
@@ -0,0 +1,166 @@
+/** @jsxImportSource @emotion/react */
+import React, { useMemo } from 'react';
+import { Typography } from '@mui/material';
+import { EllipseText, Table, TableProps, Token } from 'components';
+import { generateBscScanUrl, getTokenIdFromVAddress } from 'utilities';
+import { formatCoinsToReadableValue } from 'utilities/common';
+import { useTranslation } from 'translation';
+import { TokenId, Transaction } from 'types';
+import { useStyles } from './styles';
+
+export interface IHistoryTableProps {
+  transactions: Transaction[];
+}
+
+export const HistoryTableUi: React.FC<IHistoryTableProps> = ({ transactions }) => {
+  const { t } = useTranslation();
+  const styles = useStyles();
+
+  const columns = useMemo(
+    () => [
+      { key: 'id', label: t('history.columns.id'), orderable: true, align: 'left' },
+      { key: 'type', label: t('history.columns.type'), orderable: true, align: 'left' },
+      { key: 'txnHash', label: t('history.columns.txnHash'), orderable: true, align: 'left' },
+      { key: 'block', label: t('history.columns.block'), orderable: true, align: 'left' },
+      { key: 'from', label: t('history.columns.from'), orderable: true, align: 'left' },
+      { key: 'to', label: t('history.columns.to'), orderable: true, align: 'left' },
+      { key: 'amount', label: t('history.columns.amount'), orderable: true, align: 'right' },
+      { key: 'createdAt', label: t('history.columns.createdAt'), orderable: true, align: 'right' },
+    ],
+    [],
+  );
+
+  const cardColumns = useMemo(() => {
+    const newColumns = [...columns];
+    const [liquidityCol] = newColumns.splice(5, 1);
+    newColumns.splice(3, 0, liquidityCol);
+    return newColumns;
+  }, [columns]);
+
+  // Format assets to rows
+  const rows: TableProps['data'] = transactions.map(txn => [
+    {
+      key: 'id',
+      render: () => <Typography variant="small2">{txn.id}</Typography>,
+      value: txn.id,
+      align: 'left',
+    },
+    {
+      key: 'type',
+      render: () => (
+        <Token
+          variant="small2"
+          css={styles.whiteText}
+          symbol={getTokenIdFromVAddress(txn.vTokenAddress) as TokenId}
+        />
+      ),
+      value: txn.vTokenAddress,
```

Comment:
I think the designs for this are incorrect. The name of the column is "Type" yet we only display the token logo and name for it.
In the current app, we display the token logo and the transaction type:
<img width="267" alt="Screen Shot 2022-05-27 at 10 08 24" src="https://user-images.githubusercontent.com/44675210/170668923-381b1975-02ca-43df-bb1d-9906bb2cbee6.png">

I'll ask the design team about that

Reply:
I saw you made the change. I think that's a good call anyway (otherwise naming the column "Type" doesn't make sense). Just one thing: the token logo and the text need to be vertically aligned:
<img width="110" alt="Screen Shot 2022-05-27 at 17 11 35" src="https://user-images.githubusercontent.com/44675210/170738006-70672c4b-a07a-405c-a966-77df9bfe7ae0.png">